includes/wikia/nirvana/WikiaView.class.php

```diff
@@ -120,7 +120,6 @@ protected function buildTemplatePath( $controllerClass, $methodName, $forceRebui
 			$dirName = $this->getTemplateDir( $controllerClass );
 			foreach ( $templates as $templateName ) {
```

Comment:
How many page views and how many iterations over the loop are you expecting here? File-system scans are sub-optimal, I think. The contents of template directory  is not a subject to change between code releases. Perhaps it's worth caching or even static registering all the templates in an array?